publishable/assets/css/app.rtl.css

```diff
@@ -0,0 +1,29 @@
+@charset "UTF-8";/*!
```

Comment:
How is this file generated? Or did you just copy app.css and rename it?  If the latter, you can't do that since it won't get any future changes

Reply:
i useda npm package named rtlcss 
https://www.npmjs.com/package/rtlcss

Reply:
Right, but that's a one-time rewrite, right? I don't see you adding it as a dependency anywhere.

We'll need to make sure that file gets generated from mix before I can accept this.

Reply:
it's a npm package and installed globally  so i cant use as dependency , so your suggestion is  changing on available app.scss file ?

Reply:
I'm saying that this needs to be automated in some way, so that the rtl file can be generated any time someone updates the normal file (via compiling the raw scss)

Reply:
right, ok i'll do it , thanks!